setup.py

```diff
@@ -56,6 +56,7 @@ def has_environment_marker_platform_impl_support():
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
+        # 'Programming Language :: Python :: 3.8',  not supported yet
```

Comment:
```suggestion
        'Programming Language :: Python :: 3.8',
```

Reply:
Kmike said above that ye agreed on removing 3.8 from setup for now?

Reply:
@kmike, @wRAR and me had a discussion after that comment from @kmike, and we decided to deprecate the LevelDB storage backend in 1.8 and remove it in Scrapy 2.0, after which delaying the `setup.py` update is no longer necessary.

Reply:
Thanks for clarifying, will have these changes pushed shortly!